generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/ReductiveRowSpecGenerator.java

```diff
@@ -39,6 +39,7 @@ public ReductiveRowSpecGenerator(
         Map<Field, FieldSpec> fieldSpecsPerField = getFieldSpecsForAllFixedFieldsExceptLast(reductiveState, constraintNode);
 
         if (fieldSpecsPerField.values().stream().anyMatch(fieldSpec -> fieldSpec == FieldSpec.Empty)){
+            this.monitor.unableToEmitRowAsSomeFieldSpecsAreEmpty(reductiveState, fieldSpecsPerField);
```

Comment:
If a field spec was found to be empty, i.e. there is a field that is fixed to a set of constraints that are now contradictory, then we shouldn't emit a row.
We now call a method on the monitor to highlight the scenario